snapshot/src/test/java/com/schibsted/snapshot/WarpSpinnerTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.schibsted.snapshot
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.HtmlReportWriter
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.SnapshotVerifier
+import com.android.ide.common.rendering.api.SessionParams
+import com.android.resources.NightMode
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import com.schibsted.nmp.warp.components.WarpSpinner
+import com.schibsted.nmp.warp.components.WarpSpinnerSize
+import com.schibsted.nmp.warp.theme.WarpTheme
+import com.schibsted.nmp.warp.theme.WarpTheme.dimensions
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(TestParameterInjector::class)
+class WarpSpinnerTest(
+    @TestParameter val flavor: Flavor,
+    @TestParameter val nightMode: NightMode,
+    @TestParameter(valuesProvider = FontScaleProvider::class) private val fontScale: Float,
```

Comment:
Can remove fontscale param here, no need to have double screenshots if there is no text :)